module/VuFind/src/VuFind/Session/PluginManager.php

```diff
@@ -64,6 +67,12 @@ class PluginManager extends \VuFind\ServiceManager\AbstractPluginManager
         'VuFind\Session\Memcache' => 'Zend\ServiceManager\Factory\InvokableFactory',
     ];
 
+    protected $delegators = [
```

Comment:
For the sake of PHPDoc, we should add a comment on this property -- it can probably be copied or inherited from the parent class.

Reply:
Copied doc comment from zends service manager class.